tools/matchms/macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@TOOL_VERSION@">0.14.0</token>
+    <token name="@TOOL_VERSION@">0.15.0</token>
```

Comment:
Is the neutral losses cosine added in 0.15.0? If so, please state the version bump in the PR description